holoviews/util/__init__.py

```diff
@@ -900,8 +900,10 @@ def __call__(self, map_obj, **params):
         callback = self._dynamic_operation(map_obj)
         streams = self._get_streams(map_obj, watch)
         if isinstance(map_obj, DynamicMap):
-            dmap = map_obj.clone(callback=callback, shared_data=self.p.shared_data,
-                                 streams=streams)
+            params = {'shared_data': self.p.shared_data}
```

Comment:
Is there any reason not to put all kwargs in the params dictionary?

Reply:
Nope, I was planning to do that when we were working together on it but ran out of time.